2024-01-21 04:42:12,604 INFO    MainThread:98808 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-21 04:42:12,604 INFO    MainThread:98808 [wandb_setup.py:_flush():76] Configure stats pid to 98808
2024-01-21 04:42:12,604 INFO    MainThread:98808 [wandb_setup.py:_flush():76] Loading settings from /Users/suhuangyuan/.config/wandb/settings
2024-01-21 04:42:12,604 INFO    MainThread:98808 [wandb_setup.py:_flush():76] Loading settings from /Users/suhuangyuan/DRA/wandb/settings
2024-01-21 04:42:12,604 INFO    MainThread:98808 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/Users/suhuangyuan/DRA/main.py', 'program': '/Users/suhuangyuan/DRA/main.py'}
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_init.py:_log_setup():526] Logging user logs to /Users/suhuangyuan/DRA/wandb/run-20240121_044212-ll27i91p/logs/debug.log
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/suhuangyuan/DRA/wandb/run-20240121_044212-ll27i91p/logs/debug-internal.log
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_init.py:init():566] calling init triggers
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_init.py:init():616] starting backend
2024-01-21 04:42:12,605 INFO    MainThread:98808 [wandb_init.py:init():620] setting up manager
2024-01-21 04:42:12,605 INFO    MainThread:98808 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-01-21 04:42:12,606 INFO    MainThread:98808 [wandb_init.py:init():628] backend started and connected
2024-01-21 04:42:12,608 INFO    MainThread:98808 [wandb_init.py:init():720] updated telemetry
2024-01-21 04:42:12,637 INFO    MainThread:98808 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-01-21 04:43:43,028 ERROR   MainThread:98808 [wandb_init.py:init():779] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-01-21 04:43:43,108 ERROR   MainThread:98808 [wandb_init.py:init():1194] Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/Users/suhuangyuan/miniconda3/envs/resource/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1176, in init
    run = wi.init()
  File "/Users/suhuangyuan/miniconda3/envs/resource/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 785, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-01-21 04:43:44,121 WARNING MsgRouterThr:98808 [router.py:message_loop():77] message_loop has been closed
